package.json

```diff
@@ -6,10 +6,17 @@
   "scripts": {
     "start": "NODE_ENV=development webpack-dashboard -- webpack-dev-server --mode development --hot",
     "build": "rimraf ./dist && NODE_ENV=production webpack --mode production",
-    "eslint": "eslint . --ext .ts,.tsx,.js",
-    "eslint-fix": "eslint --fix . --ext .ts,.tsx,.js",
+    "lint": "eslint src --ext .ts,.tsx,.js",
+    "lint:fix": "eslint --fix . --ext .ts,.tsx,.js",
     "stats": "NODE_ENV=production webpack --mode production --env.stats --profile --json > stats.json && webpack-bundle-analyzer ./stats.json ./dist",
-    "test": "echo Testing... && echo OK"
+    "test": "echo Testing... && echo OK",
+    "verify": "npm run lint && npm run test"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run verify",
```

Comment:
Do you need the same hook for commit and push?

Reply:
I'll leave only the pre-commit for now